pullreq_pipeline.go

```diff
@@ -80,7 +80,7 @@ func deleteStaleGitlabPRBranch(log *logrus.Entry, pr *github.PullRequestEvent, c
 		return nil
 	}
 	repoName := pr.GetRepo().GetName()
-	repoOrg := pr.GetRepo().GetOrganization().GetLogin()
+	repoOrg := *pr.Organization.Login
```

Comment:
Question:
Why not `GetOrganization().GetLogin()` ?

Reply:
Because it just doesn't work, returns empty string. I guess we could fix upstream (or update and check the code), but currently we do it this way for `getRemoteURLGitLab` so this commit only aligns the two parts of the same feature (create pr branch, delete pr branch)

Reply:
Oh! I miss-read the code! Now that I see Fabio's comment on the same, I understand what you mean. Let's try that, then.